uniffi_bindgen/src/bindings/kotlin/mod.rs

```diff
@@ -18,11 +18,11 @@ pub use gen_kotlin::{Config, KotlinWrapper};
 use super::super::interface::ComponentInterface;
 
 pub fn write_bindings(
+    config: &Config,
```

Comment:
This file is pretty much all plumbing, feeding the config from the callers down to `gen_kotlin`.